CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### v0.33.1
+###### Features
+- Add Python 3.9 testing (pull #[1584](https://github.com/aws-cloudformation/cfn-python-lint/pull/1584))
```

Comment:
```suggestion
```

[might still be issues with Python 3.9 not caught by the tests](https://github.com/aws-cloudformation/cfn-python-lint/issues/1520) and not added here yet:

https://github.com/aws-cloudformation/cfn-python-lint/blob/4b65edb615ab9e960d117219ffc619209baa5084/setup.py#L80-L81

so maybe we should hold off on mentioning 3.9?